src/scripts/build/uploadAutoCompleteLocations.js

```diff
@@ -325,7 +324,13 @@ try {
   writeFileSync(LOCATIONS_DATA_FILE, JSON.stringify(locationDataFinal));
 
   let readStream = fs.createReadStream(LOCATIONS_DATA_FILE);
-  await uploadToS3(`autocomplete/${LOCATIONS_DATA_FILE}`, readStream);
+
+  if (process.argv.includes("--skip-upload")) {
+    console.debug("Skipping S3 upload");
+  } else {
+    const uploadToS3 = import("../s3.js");
+    await uploadToS3(`autocomplete/${LOCATIONS_DATA_FILE}`, readStream);
```

Comment:
I think this may be missing an `await` - did it work for you?

```suggestion
    const uploadToS3 = await import("../s3.js");
    await uploadToS3(`autocomplete/${LOCATIONS_DATA_FILE}`, readStream);
```

Reply:
Good catch, I only tested the `--skip-upload` branch locally...